config/rootrc.in

```diff
@@ -438,19 +438,16 @@ ACLiC.Linkdef:          _linkdef
 # Default login name (if not defined is taken from $(HOME)).
 #UsrPwd.Login:           qwerty
 #Krb5.Login:             qwerty@LOCAL.DOM.AIN
-#Globus.Login:           cd:~/.globus cf:usercert.pem  kf:userkey.pem  ad:/etc/grid-security/certificates
 #UidGid.Login:           qwerty
```

Comment:
References to UidGid should be removed in here too.

Reply:
Removed references to `UidGid` authentication.